src/main/python/ttconv/imsc/attributes.py

```diff
@@ -309,10 +310,7 @@ def to_time_format(context: TemporalAttributeWritingContext, time: Fraction) ->
   return to_frames(context, time)
 
 def to_hh_mm_ss_ms(time: Fraction) -> str:
-  millisecond = (time % 1) * 1000
-  minute, second = divmod(time, 60) 
-  hour, minute = divmod(minute, 60) 
-  return f"{hour:02}:{minute:02}:{int(second):02}.{int(millisecond):03}"
+  return str(TimeCode.from_seconds(time))
```

Comment:
`TimeCode` should be `ClockTime` (using TTML terminology) since _HH:MM:SS:mmm_ is not a SMPTE timecode